src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -92,11 +109,22 @@ public void evaluate() {
 
 	@Retention(RetentionPolicy.RUNTIME)
 	@Target({ElementType.METHOD})
+	@Repeatable(ConditionalIgnoreList.class)
 	public @interface ConditionalIgnore {
-		//todo Multiply usage with skip if any of rule say to skip
+		/**
+		 * Link to {@link IgnoreCondition} implementation.
+		 *
+		 * @return Class with {@link IgnoreCondition} implementation
+		 */
 		Class<? extends IgnoreCondition> condition();
 	}
 
+	@Retention(RetentionPolicy.RUNTIME)
+	@Target({ElementType.METHOD})
+	@interface ConditionalIgnoreList {
+		ConditionalIgnore[] value();
+	}
```

Comment:
\[Checkstyle\] WARNING: 'annotation def rcurly' has incorrect indentation level 8, expected level should be 2\.